addons/mail/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -1890,11 +1890,15 @@ QUnit.test('new messages separator [REQUIRE FOCUS]', async function (assert) {
 QUnit.test('restore thread scroll position', async function (assert) {
     assert.expect(6);
     // channels expected to be rendered, with random unique id that will be referenced in the test
-    this.data['mail.channel'].records.push({ id: 11 }, { id: 12 });
+    this.data['mail.channel'].records.push(
+        { id: 11, seen_message_id: 35},
+        { id: 12, seen_message_id: 124},
```

Comment:
Remove the seen/not seen notion from the test as it can make it fail sometimes (kind of a race condition due to focus and markasseen in tests, does not happen in real conditions). Adding these make all messages seen so the test is basically switching between two read conversations.

Reply:
I would be ok to accept this just to merge the PR but this is just hiding the problem instead of fixing it, so it should be investigated separately and added a comment to explain why + task.

also lint: space before }